mysql-test/suite/rpl/t/bug84415.test

```diff
@@ -0,0 +1,265 @@
+-- source include/master-slave.inc
```

Comment:
General concerns about stability of this test: it relies a lot on delays, which may show unexpected results in highly load environments.

Try running locally
`./mtt --debug-server --parallel=16 --repeat=128 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415 bug84415`